src/index.css

```diff
@@ -5,4 +5,9 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+}
+
+.leaflet-container {
+  width: 100%;
+  height: 100vh;
```

Comment:
The idea behind making `.leaflet-container` a top-level class is to make `height` set to 100%, which then allows manipulating the actual height by a container element wherever the map gets embedded. See https://github.com/Lifespark-Technologies/Infomed/pull/15#discussion_r419996570. Please make it 100%, and modify the styles in place where map is embedded.

Thanks!

Reply:
I've effected the suggested changes in a new  commit to this branch, please review.